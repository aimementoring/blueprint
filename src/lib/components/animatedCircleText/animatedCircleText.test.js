import React from 'react';
import renderer from 'react-test-renderer';
import AnimatedCircleText from './animatedCircleText.js';

describe('AnimatedCircleText', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(
        <AnimatedCircleText
          text="Testing my circle"
          size={200}
          duration={8}
          fontSize={20}
          reverse
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
