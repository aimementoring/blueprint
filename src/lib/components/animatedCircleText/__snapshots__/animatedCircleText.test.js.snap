// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimatedCircleText renders properly 1`] = `
<div
  className="theme-portal"
>
  <div
    className="animatedContainer "
  >
    <div
      className="circleContainer"
      style={
        Object {
          "animationDirection": "reverse",
          "animationDuration": "8s",
          "height": "200px",
          "width": "200px",
        }
      }
    >
      <div
        className="theme-portal"
      >
        <span
          className="blockTitle "
        >
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(20deg)",
              }
            }
          >
            T
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(40deg)",
              }
            }
          >
            e
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(60deg)",
              }
            }
          >
            s
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(80deg)",
              }
            }
          >
            t
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(100deg)",
              }
            }
          >
            i
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(120deg)",
              }
            }
          >
            n
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(140deg)",
              }
            }
          >
            g
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(160deg)",
              }
            }
          >
             
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(180deg)",
              }
            }
          >
            m
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(200deg)",
              }
            }
          >
            y
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(220deg)",
              }
            }
          >
             
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(240deg)",
              }
            }
          >
            c
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(260deg)",
              }
            }
          >
            i
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(280deg)",
              }
            }
          >
            r
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(300deg)",
              }
            }
          >
            c
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(320deg)",
              }
            }
          >
            l
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(340deg)",
              }
            }
          >
            e
          </span>
          <span
            className=""
            style={
              Object {
                "fontSize": "20px",
                "height": "100px",
                "transform": "rotate(360deg)",
              }
            }
          >
             
          </span>
        </span>
      </div>
    </div>
  </div>
</div>
`;
